### PR TITLE
contenteditable adds element to tabbing sequence

### PR DESCRIPTION
**Pull request by estelle**

### CODE REVIEW

File: files/en-us/web/html/global_attributes/contenteditable/index.md [hunk 1]
```diff
@@ -32,6 +32,8 @@ Note that although its allowed values include `true` and `false`, this attribute
 
 You can set the color used to draw the text insertion {{Glossary("caret")}} with the CSS {{cssxref("caret-color")}} property.
 
+Elements made editable with their own `contenteditable` attribute are focusable and participate in sequential keyboard navigation. Contenteditable elements nested within them do not get automatically added to the tabbing sequence, but can be added to the keyboard navigation sequence with [`tabindex="0'`](/en-US/docs/Web/HTML/Global_attributes/tabindex).
```
Comment (estelle) [inline files/en-us/web/html/global_attributes/contenteditable/index.md:35]:
```suggestion
Elements made editable with their own `contenteditable` attribute are focusable and participate in sequential keyboard navigation. Contenteditable elements nested within them do not get automatically added to the tabbing sequence, but can be added to the keyboard navigation sequence with [`tabindex="0'`](/en-US/docs/Web/HTML/Global_attributes/tabindex).
```

File: files/en-us/web/html/global_attributes/contenteditable/index.md [hunk 2]
```diff
@@ -32,6 +32,8 @@ Note that although its allowed values include `true` and `false`, this attribute
...
```
Comment (estelle) [inline files/en-us/web/html/global_attributes/contenteditable/index.md:35]:
```suggestion
Elements made editable, and therefore interactive, with their own `contenteditable` attribute are focusable and participate in sequential keyboard navigation, but `contenteditable` elements nested within other `contenteditable` elements are not added to the tabbing sequence. They can be added to the keyboard navigation sequence with [`tabindex="0'`](/en-US/docs/Web/HTML/Global_attributes/tabindex).
```

File: files/en-us/web/html/global_attributes/contenteditable/index.md [hunk 3]
```diff
@@ -32,6 +32,8 @@ Note that although its allowed values include `true` and `false`, this attribute
 
 You can set the color used to draw the text insertion {{Glossary("caret")}} with the CSS {{cssxref("caret-color")}} property.
 
+Elements made editable, and therefore interactive, with their own `contenteditable` attribute are focusable and participate in sequential keyboard navigation, but `contenteditable` elements nested within other `contenteditable` elements are not added to the tabbing sequence. They can be added to the keyboard navigation sequence with [`tabindex="0'`](/en-US/docs/Web/HTML/Global_attributes/tabindex).
```
Comment (dipikabh) [inline files/en-us/web/html/global_attributes/contenteditable/index.md:35]:
```suggestion
Elements that are made editable, and therefore interactive, by using the `contenteditable` attribute can be focused. They participate in sequential keyboard navigation. However, elements with the `contenteditable` attribute nested within other `contenteditable` elements are not added to the tabbing sequence by default. You can add the nested `contenteditable` elements to the keyboard navigation sequence by specifying the `tabindex` value ([`tabindex="0"`](/en-US/docs/Web/HTML/Global_attributes/tabindex)).
```

**Review by dipikabh — CHANGES_REQUESTED** [review]
Thanks for adding this clarification, @estelle!

**Review by dipikabh — APPROVED** [review]
Thanks for the updates.

**Pull request merged**